src/Repositories/Behaviors/HandleRevisions.php

```diff
@@ -138,7 +141,7 @@ public function hydrateBrowser($object, $fields, $relationship, $positionAttribu
      * @param \A17\Twill\Models\Model|null $model
      * @return void
      */
-    public function hydrateOrderedBelongsTomany($object, $fields, $relationship, $positionAttribute = 'position', $model = null)
+    public function hydrateOrderedBelongsToMany($object, $fields, $relationship, $positionAttribute = 'position', $model = null)
```

Comment:
We can clean this up in Twill 3, but for now I think we need to keep the wrongly cased method available in the public API, for case sensitive environments. We can keep the fixed method name and provide the wrongly cased as a deprecated method that calls the new one.

Reply:
@ifox My bad, I was not aware of this. I was under the impression that all methods and functions were case-insensitive in PHP. I will revert the change.

Reply:
You're right, ignore me. PHP weirdness, ha. Variables are case sensitive, but classes and functions are not. I wonder however, not in this case specifically, if Laravel itself would be affected. I could swear I already ran into issues with class names and case sensitivity.